user-guide/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/MaintenanceSettings_xml.md

```diff
@@ -75,6 +75,7 @@ This is an example of a *MaintenanceSettings.xml* file:
       <SkipRTE process="SLDataMiner.exe" thread="Database cleaning" />
     </FailoverOnRTE>
   </WatchDog>
+  <Network bitRateTracking="true">
```

Comment:
Is Network not missing a closing tag? Feel free to reject this suggestion if this is not the case :)

```suggestion
  </Network bitRateTracking="true">
```

Reply:
Hi Edith, you are right, I did forget the closing tag. However, it is not an open-close pair, so the tag goes at the end, pushed it as a new commit since I can't edit yours.